src/app/screens/confirmBtcTransaction/index.tsx

```diff
@@ -140,7 +140,7 @@ function ConfirmBtcTransaction() {
       setShowOrdinalsDetectedAlert(true);
     } else if (isLedgerAccount(selectedAccount)) {
       const txType: LedgerTransactionType = 'BTC';
-      navigate('/confirm-ledger-tx', { state: { amount: new BigNumber(amount), recipient: recipient[0], type: txType } });
+      navigate('/confirm-ledger-tx', { state: { recipient, type: txType } });
```

Comment:
I guess `recipient` should also be changed to `recipients` here as it's the expected name in location.state object in `src/app/screens/ledger/confirmLedgerTransaction/index.tsx`